streamlit-cookbook/app.py

```diff
@@ -0,0 +1,68 @@
+import openai
+import streamlit as st
+from honeyhive import HoneyHiveTracer, enrich_session
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def init_honeyhive(update=False):
+    if 'tracer' not in st.session_state or update:
+        st.session_state.tracer = HoneyHiveTracer.init(
+            api_key=os.getenv('HONEYHIVE_API_KEY'),
+            project=os.getenv('HONEYHIVE_PROJECT'),
+            session_name="Streamlit Session",
+            server_url=os.getenv('HONEYHIVE_SERVER_URL')
```

Comment:
I think we moved staging to prod, and users won't know the staging URL, so best to remove it?

Reply:
cc @dhruv-hhai

Reply:
I can remove it if we think it's best, but in the README I point the reader to `HONEYHIVE_SERVER_URL=https://app.honeyhive.ai`